### DISCUSSION TITLE
Update/Add shared subscription strategy

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx#10949] [Q&A] [answered]
I am not abe to Update/Add shared subscription group strategy on the fly when the cluster is running, is there any way to do this?

**Comment by lafirest** [comment]
@alexa22013 
Thanks for your feedback, it seems there is no way to hot-upgrade the strategy. We will discuss and implement this if necessary.

**Reply by alexa22013** [reply]
@lafirest thanks, it will really be helpful to have this

**Comment by zmstone** [comment, marked as answer]
Hi @alexa22013 

This per-group config is currently hidden in v5 because we do not want to add more users to depend on it.

We are considering to add a feature to support something like `$share/$strategy/round_robin/<toppic>/...` in future releases but no ETA yet.
When this feature is in place, we'll start deprecating this config.
